src/Combination.StringPools/Utf8StringPool.cs

```diff
@@ -90,6 +91,9 @@ PooledUtf8String IUtf8StringPool.Add(ReadOnlySpan<char> value)
     PooledUtf8String IUtf8StringPool.Add(ReadOnlySpan<byte> value)
         => AddInternal(value);
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining | MethodImplOptions.AggressiveOptimization)]
+    public static int GetAllocationSize(int length) => length + 1 + (BitOperations.Log2((uint)length) / 7);
```

Comment:
why is it adding this expression here? Is it some arbitrary extra space?

Reply:
This predicts the size of the variable-int string length prefix